tests/behavior/utils.rs

```diff
@@ -36,11 +36,13 @@ pub fn init_service(scheme: Scheme, random_root: bool) -> Option<Operator> {
         .filter_map(|(k, v)| {
             k.to_lowercase()
                 .strip_prefix(&prefix)
-                .map(|k| (k.to_string(), v))
+                .map(|k| (k.to_string(), v.to_lowercase()))
```

Comment:
We should not touch value.

Reply:
For example, users input a secret key: `XxxxxxYxxxx`.

Reply:
> For example, users input a secret key: `XxxxxxYxxxx`.

I realized this, thanks for you correction!